pkg/fileutils/fileutils.go

```diff
@@ -94,7 +94,18 @@ func Link(src string, linkName string) error {
 
 // SymbolicLink creates target as a symbolic link to src.
 func SymbolicLink(src string, target string) error {
-	// TODO Add verifications.
+	if !PathExist(src) {
+		return fmt.Errorf("Symlink %s to %s: error, src no such file or directory", target, src)
```

Comment:
when constructing an error, we usually use `failed to do sth: <detailed failing reason>`, in this case:
```
fmt.Errorf("failed to symlink %s to %s: src no such file or directory", target, src)
```

And please follow this rule in other places, thx.

Reply:
Understood it. Thanks.

Reply:
Done with #1225